perfdash/google-gcs-downloader.go

```diff
@@ -34,8 +34,8 @@ import (
 	"k8s.io/kubernetes/test/e2e/perftype"
 )
 
-// GoogleGCSDownloaderOptions is an options for GoogleGCSDownloader.
-type GoogleGCSDownloaderOptions struct {
+// DownloaderOptions is an options for Downloader.
+type DownloaderOptions struct {
 	ConfigPaths        []string
 	GithubConfigDirs   []string
 	DefaultBuildsCount int
```

Comment:
We'll probably want the `CredentialsPath` cut out of this section, its one of the GCS specific options that can go in the `NewGCSBucketUtil()`